core.js

```diff
@@ -1414,9 +1414,10 @@ async function _fromTokenizer(tokenizer) {
 	}
 }
 
-const stream = readableStream => new Promise((resolve, reject) => {
+const stream = (readableStream, sampleSize) => new Promise((resolve, reject) => {
```

Comment:
This should be an options-object. Reading `stream(someStream, 100)` at the callsite is not very dev friendly.